src/tests/specs/detailPage.spec.ts

```diff
@@ -86,6 +86,19 @@ describe('Detail View test: ', () => {
     expect(await planner.detailPage.getLinkedItems()).toContain(Workitem_Title_3);
   });
 
+  it('should remove link from workitem', async () => {
+    let linkType = 'blocks',
+      searchWorkItem = '2-Workitem_Title_3',
+      Workitem_Title_3 = 'Workitem_Title_3';
+    await planner.workItemList.openDetailPage(c.workItemTitle2);
+    await planner.waitUntilUrlContains('detail');
+    await planner.detailPage.titleInput.untilTextIsPresentInValue(c.workItemTitle2);
+    await planner.detailPage.addLink(linkType, searchWorkItem, Workitem_Title_3);
+    expect(await planner.detailPage.getLinkedItems()).toContain(Workitem_Title_3);
+    await planner.detailPage.removeLink(Workitem_Title_3);
+    expect(await planner.detailPage.getLinkedItems()).not.toContain(Workitem_Title_3);
```

Comment:
@vikram-raj test is expecting linkedListItem selector which is not there once you click close button 
expect(await planner.detailPage.getLinkedItems()).not.toContain(Workitem_Title_3);
you can replace it with links Count 
    await planner.detailPage.linkCount.untilTextIsPresent("0");
    expect(await planner.detailPage.linkCount.getTextWhenReady()).toBe("0");

and add selector in workitem-quickpreview.ts -
  linkCount = new ui.Clickable(this.$('#wi-link-total'), 'work item link total');

Reply:
@Raunak1203 Resolved this issue.